odl/discr/lp_discr.py

```diff
@@ -785,6 +997,60 @@ def uniform_discr(min_corner, max_corner, nsamples,
                                    **kwargs)
 
 
+def sequence_space(shape, exponent=2.0, impl='numpy', **kwargs):
+    """Return an object mimicing the sequence space ``l^p(R^d)``.
```

Comment:
How shall we call this guy now? It's isometrically isomorphic to any discretization of a sequence space with the same number of samples, so it's not wrong to call it sequence space (maybe with `discr_` prefix). An alternative would be simply call it what it is, namely a uniform discretization with unit cells, `uniform_discr_unitcells` maybe?